tests/e2e/specs/modules/analytics/setup-with-account-no-tag.test.js

```diff
@@ -183,9 +184,10 @@ describe( 'setting up the Analytics module with an existing account and no exist
 
 					// See the selects populate
 					await expect( page ).toMatchElement( '.mdc-select__selected-text', { text: /test account a/i } );
-					// Property and profile dropdowns should select "Set up a new property/view" options because there is no property associated with the current reference URL.
+					// Property dropdown should select "Set up a new property" options because there is no property associated with the current reference URL.
```

Comment:
```suggestion
					// Property dropdown should select "Set up a new property" option because there is no property associated with the current reference URL.
```